plugins/modules/mongodb_shell.py

```diff
@@ -229,6 +229,7 @@ def transform_output(output, transform_type, split_char):
         else:
             tranform_type = "raw"
     if transform_type == "json":
+        output = re.sub(r'\:\s*\S+\s*\(\s*(\S+)\s*\)', r':\1',output)  # Strip Extended JSON Stuff
```

Comment:
Sorry I'm checking this out after you merged.
Can you add some examples in a comment? I'm having a hard time groking what this regex matches any of the examples here: https://docs.mongodb.com/manual/reference/mongodb-extended-json/

Reply:
Oh. Something like this: `"_id" : ObjectId("58f56171ee9d4bd5e610d6b7"),`

Reply:
The shell in 5.0 decided to output extended JSON for some reason. This can't be parsed using the method in this module. So it will strip this output, i.e. NumberLong(999) -> 999